dev-utils/visualise-ui-hierarchy/README.md

```diff
@@ -0,0 +1,59 @@
+# Visualise (the) UI hierarchy (of polyPod features)
+
+## Requirements
+
+Aside from the stuff you probably already have (Node, Make, Python and Git), you
+will need [Graphviz](https://graphviz.org) and [Yarn
+1.x](https://classic.yarnpkg.com/).
+
+## Usage
+
+fhd's fork of the react-component-hierarchy tool generates a, seemingly correct,
+tree view of the component hierarchy. The little `tree2dot` script then converts
+the UTF-8-art tree into a proper graph in dot notation. Finally, Graphviz is
+used to render that graph visually as an SVG.
+
+The Makefile takes care of all this, so to just generate the SVG for
+polyExplorer, you can run:
+
+    make polyExplorer.svg
+
+If, for whatever reason, you only want the dot file, you can use:
+
+    make polyExplorer.dot
+
+And so on and so forth.
+
+## Known issues
+
+### Graph layouting
+
+Graphviz with the _fdp_ layout we're currently using seems alright, but it is
```

Comment:
nit: "The graphviz configuration"

Reply:
No, I mean Graphviz as a whole, not just that particular visualisation.